benchmark.py

```
@@ -0,0 +1,833 @@
+    
+    @property
+    def max_hash_value(self) -> int:
+        """Calculate maximum hash value based on hash_bits."""
+        return (1 << self.hash_bits) - 1
+    
+    def get_hash_space_info(self) -> str:
+        """Get human-readable hash space information."""
+        return f"{self.hash_bits}-bit hashes (0 to {self.max_hash_value:,}, ~{self.max_hash_value/1_000_000:.1f}M values)"
+
+
+def mean(data: List[float]) -> float:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)